cmd/node/main.go

```diff
@@ -308,10 +308,32 @@ func attachFileLogger(log logger.Logger, flagsConfig *config.ContextFlagsConfig)
 	return fileLogging, nil
 }
 
-func checkHardwareRequirements() error {
-	if !cpuid.CPU.Supports(cpuid.SSE4, cpuid.SSE42) {
+func checkHardwareRequirements(cfg config.HardwareRequirementsConfig) error {
+	cpuFlags, err := parseFeatures(cfg.CPUFlags)
+	if err != nil {
+		return err
+	}
+
+	if !cpuid.CPU.Supports(cpuFlags...) {
 		return fmt.Errorf("CPU Flags: Streaming SIMD Extensions 4 required")
 	}
 
 	return nil
 }
+
+func parseFeatures(features []string) ([]cpuid.FeatureID, error) {
+	flags := make([]cpuid.FeatureID, 0)
+
+	for _, cpuFlag := range features {
```

Comment:
no need for this for range, we have the `ParseFeature` function

Reply:
updated to use ParseFeature func